cmd/tnf/generate/catalog/catalog.go

```diff
@@ -129,20 +215,33 @@ func outputTestCases() {
 		return keys[i].Url < keys[j].Url
 	})
 
-	// Iterating the map by sorted identifier URL
-	for _, k := range keys {
-		fmt.Fprintf(os.Stdout, "### %s\n", identifiers.Catalog[k].Identifier.Url)
+	catalog := createPrintableCatalogFromIdentifiers(keys)
+	if catalog == nil {
+		return
+	}
+	// we need the list of suite's names
+	suites := getSuitesFromIdentifiers(keys)
+
+	// Iterating the map by test and suite names
+	for _, suite := range suites {
 		fmt.Println()
-		fmt.Println("Property|Description")
-		fmt.Println("---|---")
-		fmt.Fprintf(os.Stdout, "Version|%s\n", identifiers.Catalog[k].Identifier.Version)
-		fmt.Fprintf(os.Stdout, "Description|%s\n", strings.ReplaceAll(identifiers.Catalog[k].Description, "\n", " "))
-		fmt.Fprintf(os.Stdout, "Result Type|%s\n", identifiers.Catalog[k].Type)
-		fmt.Fprintf(os.Stdout, "Suggested Remediation|%s\n", strings.ReplaceAll(identifiers.Catalog[k].Remediation, "\n", " "))
-		fmt.Fprintf(os.Stdout, "Best Practice Reference|%s\n", strings.ReplaceAll(identifiers.Catalog[k].BestPracticeReference, "\n", " "))
+		fmt.Fprintf(os.Stdout, "### %s\n", suite)
+		fmt.Println()
+		for _, k := range catalog[suite] {
+			fmt.Fprintf(os.Stdout, "#### %s\n", k.testName)
+			fmt.Println()
+			fmt.Println("Property|Description")
+			fmt.Println("---|---")
+			fmt.Fprintf(os.Stdout, "Test Name|%s\n", k.testName)
```

Comment:
`Test Case Name`
Sometimes I get confused which catalog I am looking at :-)

Reply:
should it also be Test Case Name in the other catalogue (Test Case Building Block section)?

Reply:
done it